test/resolve/testdata/pip/requirements.txt

```diff
@@ -1,9 +1 @@
-pandas==1.5.1
 numpy==1.24.2
-pip==22.0.4
-python-dateutil==2.8.2
-pytz==2022.7.1
-setuptools==58.1.0
-six==1.16.0
-certifi==2023.7.22
-# comment
```

Comment:
Only install one dependency to speed up the alpine test

Reply:
would it be possible to find a dependency which takes less time to build? :blush:

Reply:
But then we don't test the more complex case of c-compiled dependencies in python. Do we want that?

Reply:
It is good to test a c-compiled dependency, but there ought to be a smaller one out there?